cmd/controller-manager/app/options/options.go

```diff
@@ -55,4 +56,5 @@ func (s *ControllerManagerServer) AddFlags(fs *pflag.FlagSet) {
 	fs.StringVar(&s.K8sKubeconfigPath, "k8s-kubeconfig", "./kubeconfig", "Path to k8s core kubeconfig")
 	fs.StringVar(&s.ServiceCatalogAPIServerURL, "service-catalog-api-server-url", "", "The URL for the service-catalog API server")
 	fs.StringVar(&s.ServiceCatalogKubeconfigPath, "service-catalog-kubeconfig", "./servicecatalogkubeconfig", "Path to service-catalog kubeconfig")
+	fs.Int32Var(&s.ConcurrentSyncs, "concurrent-syncs", s.ConcurrentSyncs, "The port that the controller-manager's http service runs on")
```

Comment:
i really don't see a reason to expose it.  it is just how it is done in the kube controller-manager.  i can remove it for now.  no reason to expose a knob before we need to.